content/features/featuresDeepDive/mesh/billboard.md

```diff
@@ -0,0 +1,30 @@
+---
+title: BillBoard Mode
+image: 
+description: Learn how use Mesh BillBoard Mode.
+keywords: diving deeper, meshes, billboard, camera facing
+further-reading:
+video-overview:
+video-content:
+---
+
+# Billboard Mode
+
+Billboard is a special mode for meshes, ensuring that the mesh is always facing towards the camera. This is commonly used to display information to the user, sprites or particles for instance.
+
+Example :
+```javascript
+box.billboardMode = 7;
+```
+
+Billboard Mode Values:
+
+value|Type
+-----|-------------
+0 | BILLBOARDMODE_NONE
+1 | BILLBOARDMODE_X
+2 | BILLBOARDMODE_Y
+3 | BILLBOARDMODE_Z
+4 | BILLBOARDMODE_ALL

```

Comment:
The values are 0/1/2/4/7. There's also `BILLBOARDMODE_USE_POSITION = 128`.

I think we should tell that the values are a bit field and can be combined with a **OR** (**||** operator).